tools/common/Driver.cs

```diff
@@ -967,13 +1011,31 @@ static bool XcrunFind (ApplePlatform platform, bool is_simulator, string tool, o
 			args.Add ("-f");
 			args.Add (tool);
 
-			var output = new StringBuilder ();
-			int ret = RunCommand ("xcrun", args, env.ToArray (), output);
+			var stdout = new StringBuilder ();
+			var stderr = new StringBuilder ();
+			var both = new StringBuilder ();
+			// xcrun can write unrelated stuff to stderr even if it succeeds, so we need to separate stdout and stderr.
+			// We also want to print out what happened if something went wrong, and in that case we don't want stdout
+			// and stderr captured separately, because related lines could end up printed far from eachother in time,
+			// and that's confusing. So capture stdout and stderr by themselves, and also capture both together.
+			int ret = RunCommand ("xcrun", args, env.ToArray (),
+				(v) => {
+					lock (both) {
+						both.AppendLine (v);
+						stdout.AppendLine (v);
+					}
+				},
+				(v) => {
+					lock (both) {
+						both.AppendLine (v);
+						stderr.AppendLine (v);
+					}
+				});
 
 			if (ret == 0) {
-				path = output.ToString ().Trim ();
+				path = stdout.ToString ().Trim ();
```

Comment:
`path` is still not validated
and that it could point to something that does not exists or is not accessible, e.g. permissions
and that we will issue an `MT0000` instead of a proper error with useful diagnostic information

Reply:
I can add that in a separate PR, this should really get into our xcode11.4 support branch asap, since it's shipping pretty soon.